website/docs/contributors-guide.md

```diff
@@ -56,7 +56,7 @@ We typically organize reference docs according to the following sections; we use
 
 > **Topic**
 >
-> _Topic is used to [concise explanation of relevance to RISC Zero]_
+> _Topic is used to \[concise explanation of relevance to RISC Zero]_
```

Comment:
breaks link

Reply:
this one is the only one I'm not really sure what to do? seems like a TODO that was never addressed? 🤔

Reply:
taking another look at this, I guess this was never actually a link... 

That said, let's just drop these sections entirely -- they haven't been updated in like 2 years and don't reflect the current content organization well. 

<img width="942" alt="image" src="https://github.com/user-attachments/assets/b168c846-b0c9-4adc-bd79-cb4a87aa6ad3">